### DISCUSSION TITLE
Is there a way for me to add extra headers in loki sink configuration?

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#21333] [Q&A]
Hi, 
assuming we want to add more headers like api token auth header like below to loki sink:
```
    request:
      headers:
        "api-key" : "token"
```
I think i can do above via http sink but loki sink gives me more features and etc
I would appreciate it if anyone could help or provide guidance.
similar feature request:  https://github.com/vectordotdev/vector/issues/21332

**Comment by jszwedko** [comment]
I think this is a missing feature. Thanks for opening the request!
